backend/core/src/main/java/com/feed_grabber/core/auth/exceptions/PasswordNotValidException.java

```diff
@@ -0,0 +1,7 @@
+package com.feed_grabber.core.auth.exceptions;
+
+public class PasswordNotValidException extends RuntimeException {
```

Comment:
as i understand, there is any exception handler for this exception
add exception handler for this exception